src/Makefile.am

```diff
@@ -1 +1,131 @@
-SUBDIRS = common fairness bindings plugins cmd
+SUBDIRS = common bindings plugins
+
+AM_CXXFLAGS = \
+    $(WARNING_CXXFLAGS) \
+    $(CODE_COVERAGE_CXXFLAGS)
+
+AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS) $(SQLITE_LIBS)
+
+AM_CPPFLAGS = -I$(top_srcdir) $(FLUX_CORE_CFLAGS)
+
+noinst_LTLIBRARIES = libweighted_tree.la
```

Comment:
Suggestion: I _think_ you can still keep `libweighted_tree.la` in `src/fairness` if you instead did

```
noinst_LTLIBRARIES = fairness/libweighted_tree.la
```

Then later refer to this library as `fairness_libweighted_tree.la`?

Same with the tests, etc under `src/fairness`. (It looks like you already took this approach with the files under `src/cmd`, just use the same approach here)

Otherwise, this lo

Reply:
Oh great! Thanks for letting me know about this, and sorry I didn't apply this to `libweighted_tree` at first. I just force pushed a change to this `Makefile.am` to keep the library in `src/fairness`.

Do you think I should keep the Makefiles that I've completely commented out? Should I remove them from the repository altogether?

Reply:
Remove the Makefiles you've commented out. They won't be lost forever if you are worried about losing some information, since they'll be preserved in git history.

Reply:
Got it, thanks. I just broke up that initial [WIP] commit into two; one that moves the compilation to `src/Makefile.am`, and another commit to remove all the commented out Makefiles.